docs/admin/maintenance/noble_migration.rst

```diff
@@ -5,18 +5,11 @@ SecureDrops need to be upgraded to the newer Ubuntu 24.04 (Noble)
 operating system. This process is far simpler than past upgrades
 as it has been fully automated.
 
-Administrators have two options, on the following timeline:
+Begnning April 22nd, 2025, the SecureDrop team began automatically triggering the upgrade process, in phases, across all SecureDrop instances.
 
-* **semiautomated, through April 15, 2025:** Administrators can manually trigger the upgrade and observe the process.
-* **fully automated, after April 15, 2025:** The SecureDrop team will push updates through mid- to late-April that automatically trigger the upgrade process, in phases, across all SecureDrop instances.
+If your SecureDrop has not yet been selected for an automated upgrade, you can still manually trigger a semiautomated upgrade to force the upgrade to Noble. We still recommend the semiautomated upgrade for larger instances or if you have a non-standard setup, as the upgrade will happen whenever you choose so you will already be available in case something goes wrong during the process.
```

Comment:
We're not upgrading mon servers yet, so I think we should still push the semiautomated upgrade to people:

```suggestion
You can still manually trigger a semiautomated upgrade to Noble. We still recommend the semiautomated upgrade for larger instances or if you have a non-standard setup, as the upgrade will happen whenever you choose so you will already be available in case something goes wrong during the process.
```